config/routes.rb

```diff
@@ -8,7 +8,13 @@
   resources :addresses
   resources :locations
 
-  resources :admins  
+  resources :admins do 
+    member do
+      get 'edit_custom'
+      patch 'change_password', to: 'admins#change_password', as: :changepass
```

Comment:
```suggestion
      patch 'change_password', to: 'admins#change_password'
```
change_password is meaningful than `changepass`

Reply:
changepass changed to change_password